### DISCUSSION TITLE
Possible to disable clean up of Wildcard imports?

### DISCUSSION THREAD

**Discussion by agreedSkiing** [redhat-developer/vscode-java#2990] [Q&A] [answered]
If I have organize import active in all wildcard imports are fixed to include the correct class or static method instead, is it possible to keep them as wildcards instead of expanding the imports?

**Comment by fbricon** [comment, marked as answer]
Currently the best solution is to set "java.sources.organizeImports.starThreshold" and "java.sources.organizeImports.staticStarThreshold" to a low value. But it's not perfect as it would collapse discrete imports you might want to keep into star imports

**Reply by agreedSkiing** [reply]
Going to try settings those

**Reply by agreedSkiing** [reply]
Yeah that did the trick for me :)

**Comment by agreedSkiing** [comment]
Wow I write like an idiot... What I wanted to write was "If I have organize imports active and organize imports in a java class where I want wildcard imports to reduce the amount of import lines how do I configure the organizer to not remove the wildcards? This would include static imports and normal imports."
